medicines/web/src/components/page/index.tsx

```diff
@@ -59,15 +59,17 @@ interface IPageProps {
 }
 
 const App: React.FC<IPageProps> = props => {
-  if (props.storageAllowed) {
-    TagManager.initialize({
-      gtmId: process.env.GOOGLE_GTM_CONTAINER_ID as string,
-      dataLayerName: 'dataLayer',
-    });
-    ReactGA.initialize(process.env.GOOGLE_TRACKING_ID as string, {
-      debug: true,
-    });
-  }
+  useEffect(() => {
+    if (props.storageAllowed) {
+      TagManager.initialize({
+        gtmId: process.env.GOOGLE_GTM_CONTAINER_ID as string,
+        dataLayerName: 'dataLayer',
+      });
+      ReactGA.initialize(process.env.GOOGLE_TRACKING_ID as string, {
+        debug: true,
```

Comment:
do you want to use here the `process.env.GOOGLE_DEBUG` property you introduced into the pipeline?